src/pres/loader/Filters.js

```diff
@@ -22,10 +22,12 @@ export default class User extends React.Component {
 
     constructor(props) {
         super(props)
+
+        let isAdvancedFiltersOpen = new URLSearchParams(window.location.search).get("advanced") === 'true'
+
```

Comment:
(This comment is really about line 26 `let isAdvancedFiltersOpen = new URLSearchParams(window.location.search).get("advanced") === 'true'` but I'm using two separate comment threads because that's sometimes easier)

Also, all these param checks (i.e. every time we read values from `window.location.search`) are spread out across three components:

- PGNLoader.js, where `source` (existed before this PR) is handled, and where most of these params I added are handled
- Filters.js, where `advanced` is handled
- MainContainer.js, where `variant` (existed before this PR) is handled

There could be an argument for doing these all in one place, but I don't have a strong opinion about it, and this was the simplest way to get it working

Reply:
consolidated!